ci-operator/step-registry/netobserv/install-operator/netobserv-install-operator-commands.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+update_flowcollector() {
+  FLOWCOLLECTOR=/tmp/flowcollector.yaml
+  cat <<EOF >$FLOWCOLLECTOR
+kind: FlowCollector
+apiVersion: flows.netobserv.io/v1beta2
+metadata:
+  name: cluster
+spec:
+  agent:
+    ebpf:
+      cacheActiveTimeout: 5s
+      cacheMaxFlows: 100000
+      features: []
+      sampling: 1
+    type: eBPF
+  consolePlugin:
+    logLevel: info
+    portNaming:
+      enable: true
+      portNames:
+        '3100': loki
+  deploymentModel: Direct
+  exporters: []
+  kafka:
+    address: kafka-cluster-kafka-bootstrap.netobserv
+    tls:
+      caCert:
+        certFile: ca.crt
+        name: kafka-cluster-cluster-ca-cert
+        type: secret
+      enable: false
+      userCert:
+        certFile: user.crt
+        certKey: user.key
+        name: flp-kafka
+        type: secret
+    topic: network-flows
+  loki:
+    enable: true
+    lokiStack:
+      name: loki
+    mode: Monolithic
```

Comment:
Monolithic mode is set but the spec has lokistack mode defined

Reply:
that's just a default config, it will not be used as long as mode: LokiStack is not set.